static/js/components/dashboard/CourseGrade.js

```diff
@@ -41,7 +41,7 @@ export default class CourseGrade extends React.Component {
   };
 
   renderCourseProgressLink = (courseRun: CourseRun, grade: number|string|null): React$Element<*>|null => (
-    <a href={`${EDX_LINK_BASE}${courseRun.course_id}/progress`} target="_blank">
+    <a href={`${EDX_LINK_BASE}${courseRun.course_id}/progress`} target="_blank" rel="noopener noreferrer">
```

Comment:
we could, but I don't see a strong reason to do so - they're just adding a few attributes to the tags, not altering the appearance or anything like that. If you feel strongly we should I can pull the eslint upgrades out into a separate PR.